WickedEngine/wiApplication.cpp

```diff
@@ -101,6 +101,15 @@ namespace wi
 			return;
 		}
 
+		static bool startup_workaround = false;
```

Comment:
without having the enum for device type, here you could check easily if device is dx12, for example by dynamic cast, or this bool could be set to true when the dx12 device gets created.

Reply:
"without having the enum" not sure what you mean , let me know.

Reply:
For example: 
```cpp
if(dynamic_cast<GraphicsDevice_DX12*>(device) != nullptr)
{
  workaround();
}
```

Or without dynamic cast, in wiapplication.cpp, when dx12 device gets created:
```
if(dx12)
{
  new GraphicsDevice_DX12(...)
  startup_workaround = false; // so the startup workaround will get executed
}
```

Reply:
Fixed.